Fix for Unicode error that causes one of the tests to fail

For reference, see 
http://stackoverflow.com/questions/21129020/how-to-fix-unicodedecodeerror-ascii-codec-cant-decode-byte 
and http://nedbatchelder.com/text/unipain/unipain.html

Patch created by:

Adam Huffman
The Francis Crick Institute

diff -ur sympy-sympy-1.0/sympy/utilities/autowrap.py sympy-sympy-1.0.new/sympy/utilities/autowrap.py
--- sympy-sympy-1.0/sympy/utilities/autowrap.py	2016-03-08 18:35:33.000000000 +0000
+++ sympy-sympy-1.0.new/sympy/utilities/autowrap.py	2016-05-15 10:54:52.870235379 +0100
@@ -164,7 +164,7 @@
         except CalledProcessError as e:
             raise CodeWrapError(
                 "Error while executing command: %s. Command output is:\n%s" % (
-                    " ".join(command), e.output.decode()))
+                    " ".join(command), e.output.decode('ascii', 'ignore')))
         if not self.quiet:
             print(retoutput)
 
